addons/payment_mercado_pago/models/payment_transaction.py

```diff
@@ -170,3 +178,18 @@ def _process_notification_data(self, notification_data):
             self._set_error(
                 "Mercado Pago: " + _("Received data with invalid status: %s", payment_status)
             )
+
+    @api.model
+    def _mercado_pago_response_msg(self, notification_data):
```

Comment:
https://github.com/odoo/odoo/pull/150435#discussion_r1462107265 resolved with no reason?

Reply:
oO I did change it, but must have made a ctrl+z too much I guess... Sorry about that